Review.java

```diff
@@ -165,4 +165,48 @@ public static String randomAdjective()
       return randomNegativeAdj();
     }
   }
+  public static double totalSentiment(String fileName)
+{
+String file = textToString(fileName);
+String word = "";
+String space = " ";
+double totalVal = 0.0;
+for(int i = 0; i < file.length(); i++)
+{
+String letter = file.substring(i, i + 1);
+if(letter.equals(space))
+{
+removePunctuation(word);
+totalVal += sentimentVal(word);
+word = "";
+}
+else
+{
+word += letter;
+}
+}
+removePunctuation(word);
+totalVal += sentimentVal(word);
+return totalVal;
+}
+public static int starRating(String fileName)
+{
+int totalSentiment = (int) totalSentiment(fileName);
+if(totalSentiment < 0)
+{
+return 1;
+}
+else if(totalSentiment < 5)
+{
+return 2;
+}
+else if(totalSentiment < 15)
+{
+return 3;
+}
+else
+{
+return 4;
+}
+}
 }

```

Comment:
Maybe add some indents along with your code to make it easier to read but otherwise looks good!

Reply:
Everything looks good, I would suggest indenting it so that it is easy to read